apps/functions/document-publish/publish-document/index.js

```diff
@@ -1,6 +1,57 @@
+import config from './config';
```

Comment:
Given the issue I had, this might need `.js`

Reply:
Good spot!